docs/docs/changelog/changelog.md

```diff
@@ -15,557 +15,31 @@
-* add input compressor pressures to output ([#140](https://github.com/equinor/ecalc/issues/140)) ([74e3e56](https://github.com/equinor/ecalc/commit/74e3e5673bad36bf30d8b217609819a79d7e76bb))
-* add pump results to system v2 ([8cf9e1b](https://github.com/equinor/ecalc/commit/8cf9e1b0d3ab8438291303663fc83092de1c808a))
-* add stream conditions to compressor v2 ([#194](https://github.com/equinor/ecalc/issues/194)) ([232f83b](https://github.com/equinor/ecalc/commit/232f83bf91044b706ba4c7715ceddf71f9456644))
-* add support for system v2 in FDE ([e6d1f93](https://github.com/equinor/ecalc/commit/e6d1f938d62d68479835f90932bc09b49203a6c9))
-* add support for temporal operational settings in v2 ([f2b217a](https://github.com/equinor/ecalc/commit/f2b217acaaf445df03fba077cd7407a4c37375d2))
-* check that file exist when validating yaml for run ([#388](https://github.com/equinor/ecalc/issues/388)) ([441722f](https://github.com/equinor/ecalc/commit/441722f9e8d86b4a48c1191a413f931e336d6ff7))
-* expose yaml variables ([1fe9dd0](https://github.com/equinor/ecalc/commit/1fe9dd0e3ceae658afaba24a2b18b91b3a11da43))
 * implement adjustment factors for genset ([#555](https://github.com/equinor/ecalc/issues/555)) ([4c84931](https://github.com/equinor/ecalc/commit/4c8493185440c837248daf998801ef1434298b44))
-* include rate type in header for csv export ([#279](https://github.com/equinor/ecalc/issues/279)) ([5edd0cc](https://github.com/equinor/ecalc/commit/5edd0ccff2b2f1dbfe746a666266b59c714a5eef))
-* make version comparable ([8ea5b2f](https://github.com/equinor/ecalc/commit/8ea5b2f735d9a9a6755a5e8983a322036e62dc21))
-* multiple streams in system ([#242](https://github.com/equinor/ecalc/issues/242)) ([419c2e9](https://github.com/equinor/ecalc/commit/419c2e9cef6f6bb768b5e140a5092650cacd245b))
-* new PfS columns ([#429](https://github.com/equinor/ecalc/issues/429)) ([a6b198e](https://github.com/equinor/ecalc/commit/a6b198efa45a6caf7464d2d1dc0d2509d5d76541))
-* output pump head to json-file ([#49](https://github.com/equinor/ecalc/issues/49)) ([60720f4](https://github.com/equinor/ecalc/commit/60720f429cb5da82cd839740eca8d3039c9d5969))
-* support bfs_tree in graph ([a4fff16](https://github.com/equinor/ecalc/commit/a4fff168dfa32d04588bb7e3b71de8c12e9dd6d0))
-* support name for crossover streams ([#236](https://github.com/equinor/ecalc/issues/236)) ([c801f3f](https://github.com/equinor/ecalc/commit/c801f3f0fa4b967c15c5122ac8997695f38bae12))
-* train v2 yaml and dto ([#272](https://github.com/equinor/ecalc/issues/272)) ([b0e3466](https://github.com/equinor/ecalc/commit/b0e346618bb3b39a186eb814dd40be2f7d905122))
-* validate expression reference tokens ([#404](https://github.com/equinor/ecalc/issues/404)) ([2af9d13](https://github.com/equinor/ecalc/commit/2af9d1332864b3e3bf17f84bd980b8bd436e7c7a))
-* validate model references and types ([dd8d701](https://github.com/equinor/ecalc/commit/dd8d701a5ac546407e6dcd0549991c3c73dff106))
```

Comment:
this should be included, line 52

Reply:
It is included in 8.18.1, should it still be in 8.19?

Reply:
aha, diff mismatch then, no then it should not :)